force-app/main/default/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -474,7 +476,9 @@
         con.MailingState = addr.MailingState__c;
         con.MailingPostalCode = addr.MailingPostalCode__c;
         con.MailingCountry = addr.MailingCountry__c;
-        con.Undeliverable_Address__c = addr.Undeliverable__c === undefined ? false : addr.Undeliverable__c;
+        if(con.hasOwnProperty('Undeliverable_Address__c') && addr.hasOwnProperty('Undeliverable__c')){
+            con.Undeliverable_Address__c = addr.Undeliverable__c === undefined ? false : addr.Undeliverable__c;
```

Comment:
In the event that a user doesn't have access/permissions to the Undeliverable field, its value wouldn't get copied here. Wanted to double check to make sure that we're ok with that here?

Reply:
Yeah, I'm questioning this now, but the update call on Contact will fail is this field gets changed without permissions, so we'd need to include Update access to the Undeliverable field on Contact in our permissions check.